### PR TITLE
config: enable tichi for ticdc

### PR DESCRIPTION
**Pull request by hi-rustin**
part of https://github.com/ti-community-infra/configs/issues/173

### CODE REVIEW

**Comment by hi-rustin** [comment]
/hold

- [ ] Migration SIG check config
- [ ] Give the bot admin permission
- [ ] @zhouqiang-cl turn off the sre-bot features

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -695,6 +698,32 @@ ti-community-blunderbuss:
...
       - tiancaiamao
+  - repos:
+      - pingcap/ticdc
+    pull_owners_endpoint: https://prow.tidb.io/ti-community-owners
+    max_request_count: 2
+    exclude_reviewers:
+      # Bots
+      - ti-chi-bot
+      - ti-srebot
+      # Inactive reviewers for dumpling.
```
Comment (amyangfei) [inline prow/config/external_plugins_config.yaml:709]:
```suggestion
      # Inactive reviewers for ticdc.
```

File: prow/config/external_plugins_config.yaml [hunk 2]
```diff
@@ -695,6 +698,32 @@ ti-community-blunderbuss:
...
+      - amyangfei
```
Comment (amyangfei) [inline prow/config/external_plugins_config.yaml:710]:
```suggestion
```

File: prow/config/external_plugins_config.yaml [hunk 3]
```diff
@@ -695,6 +698,32 @@ ti-community-blunderbuss:
...
+      - csuzhangxc
+      - glorv
+      - GMHDBJD
+      - IANTHEREAL
+      - leoppro
```
Comment (amyangfei) [inline prow/config/external_plugins_config.yaml:715]:
```suggestion
```

File: prow/config/external_plugins_config.yaml [hunk 4]
```diff
@@ -695,6 +698,32 @@ ti-community-blunderbuss:
...
+      - lonng
+      - WangXiangUSTC
+      - YuJuncens
+      - overvenus
+      - july2993
+      - suzaku
+      - Little-Wallace
+      - holys
+      - liuzix
```
Comment (amyangfei) [inline prow/config/external_plugins_config.yaml:724]:
```suggestion
```

File: prow/config/external_plugins_config.yaml [hunk 5]
```diff
@@ -695,6 +698,32 @@ ti-community-blunderbuss:
...
+      - amyangfei
+      - csuzhangxc
+      - glorv
+      - GMHDBJD
+      - IANTHEREAL
+      - leoppro
+      - lonng
+      - WangXiangUSTC
+      - YuJuncens
+      - overvenus
```
Comment (amyangfei) [inline prow/config/external_plugins_config.yaml:719]:
```suggestion
```

**Review by amyangfei — COMMENTED** [review]
LGTM

**Review by Mini256 — APPROVED** [review]

**Pull request merged**